code/part-one/tests.js

```diff
@@ -0,0 +1,129 @@
+const chai = require('chai');
+const expect = chai.expect;
+const { hash } = require('./utils/helpers.js');
+const { Block, Blockchain, Transaction } = require('./blockchain.js');
+
+
+// Test transaction data
+const testFromAddress = hash('testFromAddress');
+const testToAddress = hash('testToAddress');
+const testAmount = '10.00';
+const transaction = new Transaction(
+  testFromAddress,
+  testToAddress,
+  testAmount
+);
+
+// Test blockchain data
+const blockchain = new Blockchain();
+const genesisBlock = blockchain.chain[0];
+const newBlock = new Block();
+const miningAddress = hash('miningAddress');
+let timesMined = 0;
+
+// Test block data
+const testHash = '0123TESTHASH';
```

Comment:
`previousHash` or `testPreviousHash` would probably be more descriptive

Reply:
I am not really sure how to make the initial hash more descriptive. Should I just generate a hash using the hash function and place any arbitrary string in there or what do you think?

Reply:
I just meant the variable name. To me, a hash is a general thing, but a "previousHash" is a specific component of a block. I don't care what the string itself is.